tests/Core/DateTime/FluentTimePickerTests.cs

```diff
@@ -55,6 +55,19 @@ public void FluentTimePicker_TryParseValueFromString(string? value, string? time
         }
     }
 
+    [Fact]
+    public void FluentTimePicker_EmptyStringToNull()
```

Comment:
This test does not validate the correction made: it works even with the code in the `dev` branch.

You should specify a initialized `Value`.

```csharp
[Fact]
public void FluentTimePicker_ResetValue()
{
    // Arrange
    var picker = new TestTimePicker()
    {
        Value = System.DateTime.Today,   // Default date 
    };

    // Set a value
    picker.CallTryParseValueFromString("01:23", out var initialDate, out var _);
    Assert.Equal("01:23:00", initialDate?.ToString("HH:mm:ss"));

    // Reset
    picker.CallTryParseValueFromString(string.Empty, out var resultDate, out var _);
    Assert.Null(resultDate?.ToString("HH:mm:ss"));
}
```